src/PKSim.Core/Snapshots/Mappers/BaseClassifiableMapper.cs

```diff
@@ -0,0 +1,23 @@
+using System.Threading.Tasks;
+using OSPSuite.Core.Domain;
+
+namespace PKSim.Core.Snapshots.Mappers
+{
+   public abstract class BaseClassifiableMapper<T> : SnapshotMapperBase<T, Classifiable> where T : IClassifiableWrapper, new()
+   {
+      public override Task<Classifiable> MapToSnapshot(T model)
+      {
+         return SnapshotFrom(model, snapshot =>
+         {
+            snapshot.ClassificationPath = model.Parent?.Path;
+            snapshot.Name = model.Name;
+         });
+      }
+
+      public override Task<T> MapToModel(Classifiable snapshot)
+      {
+         return Task.FromResult(new T());
```

Comment:
When you deserialize the classifiable, the path and name are not really properties that can be set on the newly created model so no properties are set here.